concepts/array-analysis/introduction.md

```diff
@@ -1,8 +1,9 @@
 # Introduction
 
-In JavaScript, an array is a list-like structure with no fixed length which can hold any type of primitives or objects, or even mixed types. The array elements can be accessed by their index. Arrays are also given a bunch of built-in methods. Some of these built-in methods can analyse the contents of the array. Many of the built-in functions that analyse the contents of an array, take a [`predicate`][predicate_in_programming] as argument.
+[Arrays][arrays-concept] have built-in [methods][mdn-array-methods] to analyse the contents of the array.
+Many of the built-in functions that analyse the contents of an array, take a function that returns true or false as an argument. Such a function is called [`predicate`][predicate_in_programming].
```

Comment:
* 1 sentence per line :wink: 
* The beginning of line 4 currently repeats much of the content of line 3. Could you try to rephrase so the parts fit better together?